ui/components/app/wallet-overview/eth-overview.js

```diff
@@ -88,6 +93,40 @@ const EthOverview = ({ className }) => {
               {balanceIsCached ? (
                 <span className="eth-overview__cached-star">*</span>
               ) : null}
+              <ButtonIcon
+                data-testid="home__portfolio-site"
+                color={IconColor.primaryDefault}
+                iconName={ICON_NAMES.DIAGRAM}
+                ariaLabel={t('portfolio')}
+                size={BUTTON_ICON_SIZES.MD}
```

Comment:
As per slack conversation
```suggestion
                size={BUTTON_ICON_SIZES.LG}
```